File: FrontEnd/src/app/features/reservations/classes/models/reservation.ts
import { Guid } from 'guid-typescript'
import { Passenger } from './passenger'

export class Reservation {

    reservationId: Guid

    date: string
    refNo: string
    adults: number
    kids: number
    free: number
    totalPersons: number
    ticketNo: string
    email: string
    phones: string
    remarks: string

    destinationDescription: string
    customerDescription: string
    driverDescription: string
    pickupPointDescription: string
    portDescription: string
    shipDescription: string

    passengers: Passenger[]

}

File: BackEnd.Integration.Tests/Features/Reservations/Models/Reservation.cs
namespace BackEnd.IntegrationTests {

    public class Reservation : ReservationBase {

        public string Date { get; set; }
        public int DestinationId { get; set; }
        public int CustomerId { get; set; }
        public int PortId { get; set; }
        public int Adults { get; set; }
        public int Kids { get; set; }
        public int Free { get; set; }
        public string TicketNo { get; set; }

    }

}